panasonic_camera/camera.py

```diff
@@ -157,6 +162,14 @@ def get_info_capability(self) -> Capability:
             states=find_all_text(camrply, 'getstatelist/getstate'),
             specifications=find_all_text(camrply, 'camspeclist/camspec'))
 
+    def register_with_camera(self, device_name='robot-cameraman'):
```

Comment:
I consider to publish the module `panasonic_camera` as a package/library in the future. Therefore, it should be independent from the project. It would be confusing to use the device name `robot-cameraman` outside of this project. Hence, I would not use this default. Actually, I wouldn't use a default at all, since it is not transparent to the user, where the name comes from, if he does not know about this default value. Further, the user might like to change the device name. I suppose it should be passed by `PanasonicCameraManager`. I suggest to not use a default value in `PanasonicCameraManager` either, but make it an optional attribute `_device_name`. Only if the attribute `_device_name` is defined (not `None`), `register_with_camera` should be called. The user may define the device name as a CLI argument. Thereby, the user is responsible to know, if the camera requires the `accctrl` request or not. We can improve this later (detect automatically if `accctrl` request is required). It should be a solution we can both work with right now. What do you think?

Reply:
Sounds like a good solution, and I think I can implement that.

Reply:
So, I went ahead and did this. I realized that `_device_name` is ambiguous, so I went with `_identify_as`, and added that as an instance attribute on the PanasonicCameraManager class. I also added `identifyAs` CLI arguments to robot_cameraman and camera_manager.

Reply:
Good catch! `_device_name` is missleading in the context of `PanasonicCamera`, since it is more likely to indicate the name of the camera device. However, it is actually the name of the device that registers with the camera. Besides, I consider it as ambigous in the context of a CLI argument of `robot_cameraman` (like `--deviceName`), since the cameraman may consist to different devices (e.g. camera and gimbal). 

I am not quite happy with `--identifyAs` either, as its purpose may be unclear. We can still change its name in the future. Since it is only used/required for Panasonic Cameras, I suggest to call it `--identifyToPanasonicCameraAs`, `--registerWithPanasonicCameraAs` or even `--accessControlOfPanasonicCameraAs` for now.

Reply:
That makes sense. `--identifyToPanasonicCameraAs` is my favorite. I suppose in the future we might support other camera brands that require something similar, and if that happens we can rename to `--identifyToCameraAs`. I made the changes in 9d492f3655f4c7628dd06ce15ebd6bcb629e6a0c . I kept `_identify_as` and `identify_as` internally for brevity in the code, and used `--identifyToCameraAs` in the CLI for `panasonic_camera` with the rationale that in the context of panasonic_camera there's no need to specify the brand. Look ok?

Reply:
Looks great :+1: